apps/api-extractor/src/generators/DtsRollupGenerator.ts

```diff
@@ -179,7 +179,11 @@ export class DtsRollupGenerator {
         }
 
         if (entity.exported) {
-          replacedModifiers = 'export ' + replacedModifiers;
+          if (entity.originalName === 'default') {
+            (span.parent || span).modification.suffix = `\nexport default ${entity.nameForEmit};`;
```

Comment:
> (span.parent || span) [](start = 11, length = 22)

Could you separate this into a case for "span" vs "span.parent" and add a comment explaining what the parent would be if it exists?